airflow/providers/docker/hooks/docker.py

```diff
@@ -55,6 +55,7 @@ def __init__(
         base_url: Optional[str] = None,
         version: Optional[str] = None,
         tls: Optional[str] = None,
+        timeout: int = 60,
```

Comment:
can/should we import the default of the package rather than hard code value from our side?

https://github.com/docker/docker-py/blob/a48a5a9647761406d66e8271f19fab7fa0c5f582/docker/constants.py#L6

Reply:
@eladkal which one do you think will work best? Keeping None as default or passing the default of package by importing?

Reply:
If setting to None falls to the default within the package then I guess it's prefered.

Reply:
Ok @eladkal then i will verify it and make relevant changes

Reply:
@eladkal I have added default within package by importing it as default param.